lib/stripe/util.rb

```diff
@@ -91,47 +91,50 @@ def self.convert_to_stripe_object(data, opts = {})
 
     def self.log_error(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
         log_internal(message, data, color: :cyan,
-          level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
+                                    level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
       end
     end
 
     def self.log_info(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_INFO
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_INFO
         log_internal(message, data, color: :cyan,
-          level: Stripe::LEVEL_INFO, logger: Stripe.logger, out: $stdout)
+                                    level: Stripe::LEVEL_INFO, logger: Stripe.logger, out: $stdout)
       end
     end
 
     def self.log_debug(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_DEBUG
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_DEBUG
         log_internal(message, data, color: :blue,
-          level: Stripe::LEVEL_DEBUG, logger: Stripe.logger, out: $stdout)
+                                    level: Stripe::LEVEL_DEBUG, logger: Stripe.logger, out: $stdout)
       end
     end
 
     def self.file_readable(file)
       # This is nominally equivalent to File.readable?, but that can
       # report incorrect results on some more oddball filesystems
       # (such as AFS)
-      begin
-        File.open(file) { |f| }
-      rescue
-        false
-      else
-        true
-      end
+
+      File.open(file) { |f| }
+    rescue
+      false
+    else
+      true
```

Comment:
I don't really like the `def/rescue/end` pattern.

Reply:
I can see the argument that it obscures the presence of your `rescue` block, but IMO these aren't too bad as long as methods are kept relatively short.